tests/test_cache_layout.py

```diff
@@ -327,18 +320,18 @@ def setUpClass(cls):
         Share this valid token in all tests below.
         """
         cls._token = TOKEN
+        cls._api.token = TOKEN
         cls._api.set_access_token(TOKEN)
```

Comment:
should we then remove this?

Reply:
yes indeed, thanks for noticing. I've made the change.